features/ptp/deploy.sh

```diff
@@ -2,26 +2,48 @@
 set -uo pipefail
 BASEDIR="$(dirname "$0")"
 
-REPO_PATH=${HOME}/git/
-PTP_OPERATOR_DIR=${REPO_PATH}/ptp-operator
+PTP_DEPLOYMENT=${BASEDIR}/deploy
 
 # shellcheck disable=SC1091,SC1090
 . "${BASEDIR}/../lib/functions.sh"
 
-prereq(){
-  for command in git jq skopeo make; do
-    if [ ! -x "$(command -v ${command})" ]; then
-      die "Could not find the executable ${command} in the current PATH"
-    fi
-  done
-}
+[ -f "${BASEDIR}/myvars" ] || die "A 'myvars' file needs to be created, see the README"
+
+# shellcheck disable=SC1091,SC1090
+. "${BASEDIR}/myvars"
 
 deploy(){
-  pushd "${PTP_OPERATOR_DIR}" || die "pushd to ${PTP_OPERATOR_DIR} failed"
-  make deploy-setup
+  pushd "${PTP_DEPLOYMENT}" || die "pushd to ${PTP_DEPLOYMENT} failed"
+  oc apply -f 01_namespace.yaml
+  oc apply -f 02_operator_group.yaml
+
+  export channel=`oc get packagemanifest ptp-operator -n openshift-marketplace -o jsonpath='{.status.channels[].name}'`
+
+  if [[ ${channel} == "" ]]; then
+    die "failed to find ptp-operator channel"
+  fi
+
+  envsubst < 03_subscription.yaml | oc apply -f -
+
+  while [[ `oc -n openshift-ptp get csv --no-headers | wc -l` != "1" ]]; do
```

Comment:
I'm not sure if you can `oc wait` instead

Reply:
there is no oc wait for the object (they doesn't use conditions)